<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>564</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>966</width>
    <height>564</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>966</width>
    <height>564</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1031</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Labyrinth</string>
     </attribute>
     <widget class="s21::LabyrinthDrawer" name="labyrinthDrawer" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget { background: black; }</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>421</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="LabyrinthGrid">
       <item row="1" column="1" colspan="2">
        <widget class="QPushButton" name="newMazeButton">
         <property name="text">
          <string>Новый лабиринт</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="openFileLabyrinthButton">
         <property name="text">
          <string>Открыть файл</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="widthLabyrinthSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QPushButton" name="writeFileLabyrinthButton">
         <property name="text">
          <string>Записать файл</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="heightLabel">
         <property name="text">
          <string>Высота</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="widthLabel">
         <property name="text">
          <string>Ширина</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="heightLabyrinthSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Cave</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>431</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="CaveGrid">
       <item row="1" column="2">
        <widget class="QLabel" name="heightLabel_5">
         <property name="text">
          <string>Смерть</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="stopAutoStepButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="deathLimit_spinBox">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QPushButton" name="writeFileCaveButton">
         <property name="text">
          <string>Записать файл</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="widthLabel_2">
         <property name="text">
          <string>Ширина</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="widthCaveSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="openFileCaveButton">
         <property name="text">
          <string>Открыть файл</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="birthLimit_spinBox">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="autoStepButton">
         <property name="text">
          <string>Auto</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="chanceSpinBox">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QSpinBox" name="periodSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="heightLabel_4">
         <property name="text">
          <string>Жизнь</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="nextStepButton">
         <property name="text">
          <string>Next step</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="heightLabel_6">
         <property name="text">
          <string>Высота</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="heightCaveSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="chanceLabel">
         <property name="text">
          <string>Шанс</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="newCaveButton">
         <property name="text">
          <string>Новая пещера</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="s21::CaveDrawer" name="caveDrawer" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget { background: black; }</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>s21::LabyrinthDrawer</class>
   <extends>QWidget</extends>
   <header>Model/Drawer/drawer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>s21::CaveDrawer</class>
   <extends>QWidget</extends>
   <header>Model/Drawer/drawer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
